common/flatpak-dir.c

```diff
@@ -8469,7 +8469,10 @@ flatpak_dir_find_installed_ref (FlatpakDir   *self,
     }
 
   g_set_error (error, FLATPAK_ERROR, FLATPAK_ERROR_NOT_INSTALLED,
-               _("%s %s not installed"), opt_name ? opt_name : "*unspecified*", opt_branch ? opt_branch : "master");
+               _("%s/%s/%s not installed"),
+               opt_name ? opt_name : "*unspecified*",
+               opt_arch ? opt_arch : flatpak_get_arch (),
```

Comment:
This isn't 100% correct because `find_matching_ref` searches other supported architectures too, but I suppose there's not a great way to communicate that in this error message.